.travis.yml

```diff
@@ -0,0 +1,45 @@
+language: generic
+
+addons:
+  apt:
+    packages:
+    - gettext
+    - libcurl4-openssl-dev
+    - libicu-dev
+    - libssl-dev
+    - libunwind8
+    - zlib1g
+
+matrix:
+  include:
+    - os: linux # Ubuntu 14.04
+      dist: trusty
+      sudo: required
+      env: DOTNET_SDK_URL=https://go.microsoft.com/fwlink/?LinkID=798405 # v1.0.0-preview1
+    - os: osx # OSX 10.11
+      osx_image: xcode7.2
+      env: DOTNET_SDK_URL=https://go.microsoft.com/fwlink/?LinkID=798404 # v1.0.0-preview1
+
+before install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update               ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install openssl      ; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew link --force openssl ; fi
```

Comment:
C++ version in different repo: https://github.com/maxmind/MaxMind-DB-Reader-dotnet/blob/1701f6a/.travis.yml :)